concurrent_map.go

```diff
@@ -93,7 +132,7 @@ func (m ConcurrentMap) Get(key string) (interface{}, bool) {
 // Count returns the number of elements within the map.
 func (m ConcurrentMap) Count() int {
 	count := 0
-	for i := 0; i < SHARD_COUNT; i++ {
+	for i := 0; i < ShardCount; i++ {
 		shard := m[i]
 		shard.RLock()
 		count += len(shard.items)
```

Comment:
shard.size can be used here instead of len. or drop shard.size

Reply:
removed